### ISSUE TITLE
The training process does not converge

### ISSUE THREAD

**Issue by nebstudio** [youjin-DDAI/AttSec#3]
Hello, I ran the NetsurfP-2.0 DSSP8 training process of this code, and during the training process I found that the loss did not have a downward trend, and the acc did not have an upward trend, is there still something to be modified to the code on github, I hope to get your reply

**Comment by youjin-DDAI** [comment]
I don't exactly remember the shape of the curves, but there were no issues during the training process. And if you want to reproduce the performance exactly as shown in the paper's table, please use pretrained weights and evaluation code.
